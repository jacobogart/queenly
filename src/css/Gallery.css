/* Testing Purposes */
.Gallery {
}
/* ================ */
.Gallery {
  background-color: #635083;
  padding: 25px;
  border-radius: 20px;
  border: 10px inset #2e56578a;
}