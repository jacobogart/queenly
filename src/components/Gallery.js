import React, { Component } from 'react';
import Thumbnail from './Thumbnail.js';

// * CSS imports
import '../css/Gallery.css';

export default class Gallery extends Component {
  constructor(props) {
    super(props);
    this.state = {

    };
  }

  render() {
    return (
      <section className="Gallery">
    
      </section>
    )
  }
}