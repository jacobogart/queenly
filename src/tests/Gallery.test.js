import React from 'react';
import ReactDOM from 'react-dom';
import Gallery from '../components/Gallery.js';
import { shallow } from 'enzyme';

const selectSearchResult = jest.fn();
const bars=[];
const queens=[];
const cardType = "Show";
const cardData = {
  "category": "drag show",
  "dayOfWeek": "Sunday",
  "frequency": "weekly",
  "host": ["Chamilla Foxx"],
  "id": 2101,
  "imageURL": "https://pbs.twimg.com/media/D1Any4VWkAA86HL.jpg",
  "reoccuring": true,
  "startTime": [1400],
  "name": "Sunday Social"
}

describe('Gallery', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <Gallery 
      cardData={cardData}
      cardType={cardType}
      selectSearchResult={selectSearchResult}
      bars={bars}
      queens={queens}
      />
    );
  });

  it('should match snapshop', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should have default state', () => {
    expect(wrapper.state()).toEqual({});
  });
})