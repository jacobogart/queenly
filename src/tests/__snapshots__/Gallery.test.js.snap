// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery should match snapshop 1`] = `
<section
  className="Gallery"
>
  <Thumbnail
    imgURL="https://pbs.twimg.com/media/D1Any4VWkAA86HL.jpg"
    key="2101"
    name="Sunday Social"
    selectSearchResult={[MockFunction]}
  />
</section>
`;
